worblehat-domain/src/main/java/de/codecentric/psd/worblehat/domain/Book.java

```diff
@@ -97,7 +100,17 @@ public void setYearOfPublication(int yearOfPublication) {
 		this.yearOfPublication = yearOfPublication;
 	}
 
-	public void setBorrowing(Borrowing borrowing) {
-		this.borrowing = borrowing;
+    public Borrowing getBorrowing() {
+		return borrowing;
+	}
+
+	boolean isSameCopy(@Nonnull Book book) {
+		return getTitle().equals(book.title) && getAuthor().equals(book.author);
+	}
+
+	public void borrowNowByBorrower(String borrowerEmailAddress) {
+		if (borrowing==null) {

```

Comment:
I think there are spaces missing before and after the `==`. Looking at this I wonder whether we should integrate checkstyle or something like this. 

No blocker.

Reply:
We had Checkstyle once upon a time. There's still a checkstyle.xml in /src/sonar/checkstyle.xml for that matter...

Anyway, fixed for now.